fastrtps.repos

```diff
@@ -2,20 +2,20 @@ repositories:
     foonathan_memory_vendor:
         type: git
         url: https://github.com/eProsima/foonathan_memory_vendor.git
-        version: master
+        version: v1.3.1
     fastcdr:
         type: git
         url: https://github.com/eProsima/Fast-CDR.git
-        version: master
+        version: v1.1.0
     fastrtps:
         type: git
         url: https://github.com/eProsima/Fast-DDS.git
-        version: master
+        version: 2.11.x
     fastddsgen:
         type: git
         url: https://github.com/eProsima/Fast-DDS-Gen.git
-        version: master
+        version: v2.5.0
```

Comment:
This was updated on [v2.11.2](https://github.com/eProsima/Fast-DDS/blob/v2.11.2/fastrtps.repos#L17C22-L17C22):

```suggestion
        version: v2.5.1
```

Reply:
Good catch!